source/api/discovery/0.3/index.md

```diff
@@ -0,0 +1,784 @@
+
+The overall ordering of the Collection is from the oldest Activity as the first entry in the first page, to the most recent as the last entry in the last page. Consuming applications _SHOULD_ therefore start at the end and walk **backwards** through the list, and stop when they reach a timestamp before the time they last processed the list.
+
+Content providers _MUST_ publish an Ordered Collection at the HTTP(S) URI listed in the `id` property of the Collection.
+
+##### id
+
+The identifier of the Ordered Collection.
+
+Ordered Collections _MUST_ have an `id` property. The value _MUST_ be a string and it _MUST_ be an HTTP(S) URI. The JSON representation of the Ordered Collection _MUST_ be available at the URI.
+
+```
+{ "id": "https://example.org/activity/all-changes" }
+```
+
+##### type
+
+The class of the Ordered Collection.
+
+Ordered Collections _MUST_ have a `type` property.  The value _MUST_ be `OrderedCollection`.
+
+```
+{ "type": "OrderedCollection" }
+```
+
+##### first
+
+A link to the first Ordered Collection Page for this Collection.
+
+Ordered Collections _SHOULD_ have a `first` property.  The value _MUST_ be a JSON object, with the `id` and `type` properties.  The value of the `id` property _MUST_ be a string, and it _MUST_ be the HTTP(S) URI of the first page of items in the Collection. The value of the `type` property _MUST_ be a string, and _MUST_ be `OrderedCollectionPage`.
+
+```
+{ 
+  "first": {
+    "id": "https://example.org/activity/page-0",
+    "type": "OrderedCollectionPage"
+  }
+}
+```
+
+##### last
+
+A link to the last Ordered Collection Page for this Collection.  As the client processing algorithm works backwards from the most recent to least recent, the inclusion of `last` is _REQUIRED_, but `first` is only _RECOMMENDED_.
+
+Ordered Collections _MUST_ have a `last` property.  The value _MUST_ be a JSON object, with the `id` and `type` properties.  The value of the `id` property _MUST_ be a string, and it _MUST_ be the HTTP(S) URI of the last page of items in the Collection. The value of the `type` property _MUST_ be a string, and _MUST_ be `OrderedCollectionPage`.
+
+```
+{ 
+  "last": {
+    "id": "https://example.org/activity/page-1234",
+    "type": "OrderedCollectionPage"
+  }
+}
+```
+
+##### totalItems
+
+The total number of Activities in the entire Ordered Collection.
+
+OrderedCollections _MAY_ have a `totalItems` property.  The value _MUST_ be a non-negative integer.
+
+```
+{ "totalItems": 21456 }
+```
+
+##### seeAlso
+
+This property is used to refer to one or more documents that semantically describe the set of resources that are being acted upon in the Activities within the OrderedCollection. This would allow the OrderedCollection to refer to, for example, a [DCAT][org-w3c-dcat] description of the dataset. For OrderedCollections that aggregate activities and/or objects from multiple sources, the referenced description should describe the complete aggregation rather than an individual source.
```

Comment:
Use of "OrderedCollection" (no space) here seems inconsistent with other entries.

Reply:
Fixed by 00a403de623c4993618af80254e0d1c92ed6bb29